Readme.md

```diff
@@ -1,24 +1,55 @@
-<!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128533319/13.1.4%2B)
-[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E3583)
-[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
-<!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView - How to apply a filter to a column on the client side
+# GridView for ASP.NET Web Forms - How to apply a filter to a column on the client side
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e3583/)**
 <!-- run online end -->
 
+This example demonstrates how to use a text string from the external text box to filter a grid column on the client.
+
+![AutoFilterByColumn](AutoFilterByColumn.gif)
```

Comment:
While the animation itself is good, I can't help but notice a few issues related to the source application itself:

1. Your filter button is not aligned with anything -- neither with the search box, nor with your grid. Moreover, the button visually "stuck" to the grid. This does not look professional. I believe you can adjust your application layout better and re-capture the animation.
2. The button caption says "Filter by **CategoryName**". Why did you chose the form "CategoryName" over "Category Name"? The column's caption says "Category Name" rather than "CategoryName", and all UI names should have a direct connection to the UI entities they are referring to. I recommend you to rename this button "Filter by Category Name".

**Bottom Line** -- I recommend you to re-capture this animation taking the mentioned issues into consideration. Thanks in advance.

Reply:
The issue is resolved.